controlsfx-samples/src/main/java/org/controlsfx/samples/textfields/HelloAutoComplete.java

```diff
@@ -146,6 +183,29 @@ private void autoCompletionLearnWord(String newWord){
         return grid;
     }
 
+	// https://stackoverflow.com/a/17465261/6094756
+	private Map<String, Color> allColorsWithName() {
+        Map<String, Color> map = new HashMap<>();
+        try {
+			Class<?> clazz = Class.forName("javafx.scene.paint.Color");
+			if (clazz != null) {
+				Field[] field = clazz.getFields();
```

Comment:
No need to load class by name.. can simply do `Color.class.getFields()`

Reply:
Thank you for noticing. I should have read through the code more attentively before copy-pasting it …